docs/modules/ROOT/pages/erc1155-supply.adoc

```diff
@@ -0,0 +1,29 @@
+= ERC-1155 Supply
+
+The OpenZeppelin xref:erc1155.adoc[ERC-1155] Supply extension that adds tracking of total supply per token id.
+Useful for scenarios where Fungible and Non-fungible tokens have to be clearly identified.
+
+[[usage]]
+== Usage
+
+In order to make an xref:erc1155.adoc[ERC-1155] token with https://docs.rs/openzeppelin-stylus/0.2.0-alpha.2/openzeppelin_stylus/token/erc1155/extensions/supply/index.html[Supply] flavour,
+you need to add the following code to your contract:
+
+[source,rust]
+----
+use openzeppelin_stylus::token::erc1155::extensions::Erc1155Supply;
+
+sol_storage! {
+    #[entrypoint]
+    struct Erc1155Example {
+        #[borrow]
+        Erc1155Supply erc1155_supply;
+    }
+}
+
+#[public]
+#[inherit(Erc1155Supply)]
+impl Erc1155Example {
+    // ...
```

Comment:
Add missing stuff to the docs after applying attitude 5.

Reply:
I'm not sure what you mean by "attitude 5"?

Reply:
[I understand now](https://openzeppelin.slack.com/archives/C06R5AGFE9G/p1733396953128739?thread_ts=1733325243.812519&cid=C06R5AGFE9G). Added!